validator/attester/service_test.go

```diff
@@ -46,8 +46,9 @@ func TestLifecycle(t *testing.T) {
 	}
 	att := NewAttester(context.Background(), cfg)
 	att.Start()
-	testutil.AssertLogsContain(t, hook, "Starting service")
 	att.Stop()
+
+	testutil.AssertLogsContain(t, hook, "Starting service")
```

Comment:
Were you able to confirm that this test specifically was flakey? I wonder if there's any value in this test at all. From what I can tell, all it's checking is that `Start` and `Stop` are emitting these logs, but these logs are emitted at the very beginning of the function. What regression could this possibly catch?

Reply:
I dont think this test was specifically flakey, I figured it would be better to test for all log assertions at the end of the test. From what I can see this test just runs the `Start` and `Stop` methods to bring code coverage up, it doesn't really test for anything specifically